app/Http/Controllers/BookController.php

```diff
@@ -126,4 +127,35 @@ public function unclaimBook(int $id, Request $request)
             'message' => 'Book {$id} was not able to be returned'
         ], 500);
     }
+ 
+    public function addBook(Request $request)
+    {
+        $request->validate([
+            'title' => 'required|string|min:1|max:20',
+            'author' => 'required|string|min:1|max:20',
+            'genre_id' => 'required|int|min:1|max:10',
+            'blurb' => 'string|max:50',
+            'image' => 'string|max:250',
+            'year' => 'int|digits:4'
+        ]);
+
+        $book = new Book();
+
+        $book->title = $request->title;
+        $book->author = $request->author;
+        $book->genre_id = $request->genre_id;
+        $book->blurb = $request->blurb ?? $request->blurb;
+        $book->image = $request->image ?? $request->image;
+        $book->year = $request->year ?? $request->year;
+
+        if ($book->save()) {
+            return response()->json([
+               'message' => 'Book created',
+            ], 201);
+        }
+        if (!$book) {
```

Comment:
No need for this if statement - just return the 500 response after the save if statement.

Purely because if $book->save failed, then it means you must not have a new book, so checking is redudant